lib/PaginationEmbed.js

```diff
@@ -2,6 +2,10 @@
 
 const ReactionHandler = require('eris-reactions');
 
+/**
+ * @typedef {import('eris').EmbedBase} EmbedBase

```

Comment:
Also not sure if jsdoc will parse this correctly

Reply:
Missed this the other day, it definitely does for me, and I'm using this method for a js framework for which I'm also working on typings for